src/config/types.rs

```diff
@@ -56,13 +56,18 @@ where
 }
 
 #[allow(non_snake_case)]
-pub fn deserialize_Keysym<'de, D>(deserializer: D) -> Result<Keysym, D::Error>
+pub fn deserialize_Keysym<'de, D>(deserializer: D) -> Result<Option<Keysym>, D::Error>
 where
     D: serde::Deserializer<'de>,
 {
     use serde::de::{Error, Unexpected};
 
-    let name = String::deserialize(deserializer)?;
+    let name: Option<String> = Option::deserialize(deserializer)?;
+    if name.is_none() {
+        return Ok(None);
+    }
+
+    let name = name.unwrap();
```

Comment:
If we keep the `String::deserialize` code here and just wrap it in the end with `Some` + an appropriate `default` tag (see above), we don't need to change the config file. We can just omit `key` for entries that don't need/want it.